core/controllers/suggestion_test.py

```diff
@@ -751,9 +749,8 @@ def test_translation_suggestion_creation_with_new_images(self):
             csrf_token=csrf_token
         )
 
-        fs = fs_domain.AbstractFileSystem(
-            fs_domain.GcsFileSystem(
-                feconf.IMAGE_CONTEXT_EXPLORATION_SUGGESTIONS, exp_id))
+        fs = fs_services.GcsFileSystem(
+                feconf.IMAGE_CONTEXT_EXPLORATION_SUGGESTIONS, exp_id)
```

Comment:
```suggestion
            feconf.IMAGE_CONTEXT_EXPLORATION_SUGGESTIONS, exp_id)
```

ditto elsewhere, only use 4 spaces indent

Reply:
Done.